sound/soc/intel/boards/Kconfig

```diff
@@ -437,4 +437,20 @@ config SND_SOC_INTEL_SOF_RT5682_MACH
 	   If unsure select "N".
 endif ## SND_SOC_SOF_HDA_COMMON || SND_SOC_SOF_BAYTRAIL
 
+if SND_SOC_SOF_TIGERLAKE
```

Comment:
Minor: The commit message could just state that "Add TGL RT1308 machine driver with DMIC and HDMI support with 4 converters." It's a a new machine driver, so DMIC and HDMI are not added, the whole machine driver is.